interfaces/issue-data.ts

```diff
@@ -55,6 +56,11 @@ export interface IssueData {
   benefactors?: fundingBenefactor[];
   disputes?: Disputes[];
   payments: Payment[];
+  network?: {
```

Comment:
add _information_ to the property name so we understand that's information and not the actual _network_.